# 1/0: if the tag enters the target areas (1) or not (0)
int32 enter_target_areas

# 1/0: if the tag indicates the box is small (1) or not (0)
int32 small_box

# 1/0: if the tag stays still (1) or not (0)
int32 stay_still

# the box tag's position [x, y, z]
std_msgs/Float64MultiArray box_tag_position
